tests/test_service.py

```diff
@@ -37,6 +41,8 @@ def __iter__(self):
     assert markers == [{'value': 'test_marker'},
                        {'key': 'test_decorator_key',
                         'value': 'test_decorator_value'},
+                       {'key': 'test_decorator_key_with_multi_value',
```

Comment:
In this case, your marker will be displayed at RP as: `test_decorator_key_with_multi_value:['test_value1', 'test_value2']`. I do not think that is what you wanted to achieve...

Reply:
my apologies, I was curious and testing how it react if I pass list and accidentally pushed. 

Fixed tests and it should be good now.

`{'key': 'test_decorator_key_with_multi_value',
  'value': 'test_value1'},
  {'key': 'test_decorator_key_with_multi_value',
  'value': 'test_value2'},`